histomicstk/preprocessing/color_conversion/lab_to_rgb.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from .rgb_to_lab import _rgb2lms, _lms2lab
+
+
+# Define conversion matrices
+_lms2rgb = np.linalg.inv(_rgb2lms)
+_lab2lms = np.linalg.inv(_lms2lab)
```

Comment:
I had to do this because the roundtrip test `rgb --> lab --> rgb` was failing due to reduced precision of the earlier hard-coded conversion matrices.